### DISCUSSION TITLE
What is the `spaced-only` card type?

### DISCUSSION THREAD

**Discussion by demicuz** [reuseman/flashcards-obsidian#39] [Q&A] [answered]
Just wondering, how is it different from simple flashcards (`#card`)?

**Comment by sorenfrederiksen** [comment, marked as answer]
I believe a #card-spaced, as indicated by the [example here](https://github.com/reuseman/flashcards-obsidian/wiki#spaced-with-card-spaced-hashtag), is just a one-sided card.

It's for things like quotes, maybe a poem, for which a question-answer format might be a little forced or inappropriate.

**Reply by sorenfrederiksen** [reply]
(And yes, the name is a little odd. "All of the cards are subject to spaced-repetition, so what distinguishes this one?" one could ask. I'm not sure. The name might make sense in the context of the Anki API or Anki community jargon.)
